compiler/ilgen/OMRIlBuilder.hpp

```diff
@@ -254,6 +254,7 @@ class IlBuilder : public TR::IlInjector
    TR::IlValue *UnsignedGreaterOrEqualTo(TR::IlValue *left, TR::IlValue *right);
    TR::IlValue *ConvertTo(TR::IlType *t, TR::IlValue *v);
    TR::IlValue *UnsignedConvertTo(TR::IlType *t, TR::IlValue *v);
+   TR::IlValue *Negate(TR::IlValue *v);
```

Comment:
I can't decide whether this particular service needs some accompanying documentation. The name really says all that needs to be said. However, I don't know if this could be confused with a boolean NOT service.